src/lib.rs

```diff
@@ -84,25 +83,27 @@ fn join_with_or<TItem: fmt::Display>(list: &[TItem]) -> String {
     ret
 }
 
-struct CachedQueries(HashMap<SupportedLanguageName, OnceLock<Result<Arc<Query>, QueryError>>>);
+#[derive(Default)]
+struct CachedQueries(BySupportedLanguage<OnceLock<Result<Arc<Query>, QueryError>>>);
```

Comment:
I believe this could now be `static` if we wanted it to be since it looks like `OnceLock::new()` is `const`

But now at least we can `#[derive(Default)]` which will stack-allocate a fixed-size data structure (specifically an array)